articles/blog.md

```diff
@@ -1,6 +1,6 @@
 # Securing Open Liberty Application with Azure Active Directory via OpenID Connect
 
-Long gone are the days when you had to create your own user account management, authentication, and authorization for your web delivered software.  Instead, contemporary applications leverage these functions (Identity and Access Management, IAM for short) from an external provider.  Java EE has powerful standard abstractions for these functions and [Open Liberty](https://openliberty.io), being a full Java EE runtime, has great options for externally provided IAM.  Open Liberty also supports IAM mainstays such as [Social Media Login](https://openliberty.io/docs/ref/feature/#socialLogin-1.0.html), [SAML Web Single Sign-on](https://openliberty.io/docs/ref/feature/#samlWeb-2.0.html) and [OpenID Connect Client](https://openliberty.io/docs/ref/feature/#openidConnectClient-1.0.html). In Bruce Tiffany's blog post "[Securing Open Liberty apps and micro-services with MicroProfile JWT and Social Media login](https://openliberty.io/blog/2019/08/29/securing-microservices-social-login-jwt.html)", you have a solid example on how to use the Open Liberty Social Media Login feature to authenticate users using their existing social media credentials. In this blog post, let's take a look at another example about how to use the Open Liberty OpenID Connect Client feature to secure applications with [Azure Active Directory](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc) using the existing Java EE IAM APIs.
+Long gone are the days when you had to create your own user account management, authentication, and authorization for your web delivered software.  Instead, contemporary applications leverage these functions (Identity and Access Management, IAM for short) from an external provider.  Java EE has powerful standard abstractions for these functions and [Open Liberty](https://openliberty.io), being a full Java EE runtime, has great options for externally provided IAM.  Open Liberty also supports IAM mainstays such as [Social Media Login](https://openliberty.io/docs/ref/feature/#socialLogin-1.0.html), [SAML Web Single Sign-on](https://openliberty.io/docs/ref/feature/#samlWeb-2.0.html) and [OpenID Connect Client](https://openliberty.io/docs/ref/feature/#openidConnectClient-1.0.html). In Bruce Tiffany's blog post "[Securing Open Liberty apps and micro-services with MicroProfile JWT and Social Media login](https://openliberty.io/blog/2019/08/29/securing-microservices-social-login-jwt.html)", you have a solid example on how to use the Open Liberty Social Media Login feature to authenticate users using their existing social media credentials. In this blog post, let's take a look at another example about how to configure social login as OpenID Connect client to secure applications with [Azure Active Directory](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc) using the existing Java EE IAM APIs.
```

Comment:
In this section, Java EE is used, but the following section you talk about Jakarta EE. I think it is better to focus on one term, say Jakarta EE.